docs/tutorials/PipelinesApi101.md

```diff
@@ -17,15 +17,36 @@ In addition, the following changes are to be expected:
 
 ### Setting up PAPIv2
 
-For now the easiest way to try PAPIv2 is to migrate an existing set up from PAPIv1 (see below). After that, copy the PAPIv2 sample configuration in [cromwell.examples.conf](https://github.com/broadinstitute/cromwell/blob/develop/cromwell.examples.conf) in place of the PAPIv1 backend.
+For now the easiest way to try PAPIv2 is to migrate an existing set up from PAPIv1 (see below). After that, copy the PAPIv2 sample configuration in [cromwell.examples.conf](https://github.com/broadinstitute/cromwell/blob/develop/cromwell.example.backends/PAPIv2.conf) in place of the PAPIv1 backend.
```

Comment:
Thinking I might get rid of this altogether save the link to an example.  Migrate from Papi v1 is not great advice for a newbie.